doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1208,6 +1208,110 @@ operations will be supported under the 'v2' resource as well.
+<tr><td style="padding-right: 40px; width: 140px;">limit</td><td>Limit the number of entities to be retrieved</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">offset</td><td>Skip a number of records</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">options</td><td>
+This operation supports two options:
+
+<ul>
+  <li><code>count</code>: when used, the total number of entities is returned in the response as a HTTP header named <code>X-Total-Count</code>.</li>
+  <li><code>canonical</code>: when used, the response payload uses the "canonical form". See JSON entity representation format section
+  for details.</li>
+</td></tr>
+</table>
+
+The response payload is an Array which contains one object per matching entity. Each entity follows
+the JSON entity representation format (described in a section above).
+
+The payload may contain the following elements (all of them optional):
+
+* `entities`: a list of entites to search for. Each entity is represented by a JSON object with the following
+  elements:
+   * `id` or `idPattern` (both cannot be used at the same time): specifies the exact ID (in the case of `id`)
+     or pattern (in the case of `idPattern`) of the entities to search for. If omitted, it means "any entity
+     ID".
+   * `type`: specifies the type of the entities to search for. If omitted, it means "any entity type".
+* `attributes`: a list of attributes names to search for. If omitted, it means "any attribute".
+* `scopes`: a list of scopes to restrict the results of the query. Each scope is represented by a JSON
+   object with a `type` (a JSON string) and `value` (whose type depends on the `type` property). The
+   different available scopes are described elsewhere.
+
+Response code:
+
+* Successful operation uses 200 OK
+* Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
+
++ Parameters
+    + limit: 20 (optional, number)
+    + offset: 20 (optional, number)
+
++ Request (application/json)
+
+        {
+            "entities": [
+                {
+                    "idPattern": ".*",
+                    "type": "myFooType"
+                },
+                {
+                    "id": "myBar",
+                    "type": "myBarType"
+                }
+            ],
+            "attributes": [
+                "temperature",
+                "humidity"
+            ],
+            "scopes": [
+                {
+                    "type": "FIWARE::Filter::qString",
+                    "value": "temperature>40;colour=black,red;+status"
+                },
+                {
+                    "type": "FIWARE::Filter::Location",
+                    "value": {
+                        "circle": {
+                            "centerLatitude": "40.418889",
+                            "centerLongitude": "-3.691944",
+                            "radius": "13500"
+                        }
```

Comment:
I think the plan is to migrate to GeoJSON ...

as we don't know exactly how the GeoQuery object will be maybe we can leave out this